app/models/planning_application_policy_class.rb

```diff
@@ -12,13 +12,14 @@ class PlanningApplicationPolicyClass < ApplicationRecord
     validates :reporting_types, presence: true
   end
 
+  after_create :initialize_planning_application_policy_sections
+
   delegate :section, :name, :url, :description, to: :policy_class
-  delegate :planning_application_policy_sections, to: :planning_application
 
-  def policy_sections
```

Comment:
Trying to avoid having methods called `policy_section` etc that actually deal with `PlanningApplicationPolicySection`s.